testing/spectest/exclusions.txt

```diff
@@ -163,7 +163,7 @@ tests/minimal/eip6110/ssz_static/ContributionAndProof
 tests/minimal/eip6110/ssz_static/Deposit
 tests/minimal/eip6110/ssz_static/DepositData
 tests/minimal/eip6110/ssz_static/DepositMessage
-tests/minimal/eip6110/ssz_static/DepositReceipt
```

Comment:
I think you don't want to touch these exclusions until the spectests have been updated. Otherwise the spectest compliance report tool will complain these tests are missing

Reply:
will add back in then